### PR TITLE
Changed Dockerfile to get CLI to work

### PR DESCRIPTION
**Pull request by Cem-Harmony**
## Issue
CLI wouldn't build with the old Dockerfile

I ended up moving the environmental variables to the Dockerfile itself.

## Operational Checklist

1. **Does this PR introduce backward-incompatible changes to the on-disk data structure and/or the over-the-wire protocol?**. (If no, skip to question 8.)

    **NO**

2. **Does this PR introduce backward-incompatible changes *NOT* related to on-disk data structure and/or over-the-wire protocol?** (If no, continue to question 11.)

    **NO**

3. **Does the existing `node.sh` continue to work with this change?**
Yes.

## TODO
Add or remove default programs.

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by fxfactorial** [comment]
very good, we cant merge because it has merge conflicts. Perhaps do this, git reset --hard the Dockerfile to whatever the latest version of the Dockerfile is, then git cherry pick your commit on top, force push here and we can merge

File: Dockerfile [hunk 1]
```diff
@@ -4,49 +4,57 @@ SHELL ["/bin/bash", "-c"]
...
+RUN echo "GIMME_GO_VERSION='${GIMME_GO_VERSION}'" >> /root/.bash_profile
 
 RUN echo "GO111MODULE='on'" >> /root/.bash_profile
 
 RUN echo ". ~/.bash_profile" >> /root/.profile
 
 RUN echo ". ~/.bash_profile" >> /root/.bashrc
 
-ENV GOPATH='/root/go'
+ENV GOPATH="/root/go"
```
Comment (fxfactorial) [inline Dockerfile]: prefer single quotes for when you dont need variable interpolation

**Comment by LeoHChen** [comment]
@Cem-Harmony , please don't create branches in main repo.  Please follow the process, https://github.com/harmony-one/harmony-open/tree/master/git-cheat-sheet

**Comment by Daniel-VDM** [comment]
Jenkins, test this please

**Comment by fxfactorial** [comment]
Jenkins, test this please

**Pull request merged**